clang/docs/ClangFormatStyleOptions.rst

```diff
@@ -4226,6 +4226,19 @@ the configuration (without a prefix: ``Auto``).
   ``ClassImpl.hpp`` would not have the main include file put on top
   before any other include.
 
+.. _IncludeSortKey:
+
+**IncludeSortKey** (``IncludeSortKeyDiscriminator``) :versionbadge:`clang-format 20` :ref:`¶ <IncludeSortKey>`
+  When sorting includes in each block, sort by the specified sort key
+
+  Possible values:
+
+  * ``ISK_Path`` (in configuration: ``Path``)
```

Comment:
Maybe add examples, not everyone knows what a stem is.

Reply:
I think the comment is sufficient explanation of what `stem` means in this case?